pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -366,6 +428,53 @@ class CreateVM extends React.Component {
                            onChange={this.onChangedEventChecked.bind(this, 'startVm')} />
                     {_("Immediately Start VM")}
                 </label>
+
+                <hr />
+
+                { this.props.providerName == 'LibvirtDBus' &&
```

Comment:
Storage Pools and volumes object exist in Virsh provider as well. You don't need to limit this feature to libvirtDBus.